narwhal/primary/src/aggregators.rs

```diff
@@ -45,6 +51,10 @@ impl VotesAggregator {
         self.votes.push((author.clone(), vote.signature));
         self.weight += committee.stake(&author);
 
+        self.metrics
+            .votes_received_last_round
```

Comment:
nit: I believe having this here would update the metric every time we receive a vote. Could we just update it once at the end of the [propose_header](https://github.com/MystenLabs/sui/blob/main/narwhal/primary/src/core.rs#L366) method?

Reply:
@arun-koshy the reason why I would like to update on every vote, is because this will give us a good idea (especially in case of consensus stall) how many votes we managed to get for our header